frontend/src/reports/query/Query.svelte

```diff
@@ -18,16 +18,20 @@
   const results: Record<string, Result<QueryResult, string>> = {};
   const is_open: Record<string, boolean> = {};
 
-  onMount(() =>
+  onMount(() => {
```

Comment:
The return value is important so that unsubscribe is called on unmount. You can simply add a second onMount block